libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -290,6 +290,24 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to enable PreciseAnswer Generation. 
```

Comment:
Teh "G" on generation should not be capitalized.

Reply:
Updated as per comment

---
In reply to: [436231170](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r436231170) [](ancestors = 436231170)